paper/writeup.tex

```diff
@@ -275,7 +285,8 @@ \section{Introduction}
 we define and describe how to compute new measures of (dis-)agreement between ARGs
 that are motivated by the Robinson-Foulds distance between trees
 but account for haplotype identity.
-These measures show that the vast majority of these extended haplotypes are correct,
+These measures show that the vast majority of these extended haplotypes are correct, % NSP-22NOV: "vast majority" is correct in simulated (but not inferred) ARGs, right?
+% HF-24NOV : It should be both, as the IE line in Figure 6D shows that inferred trees match the TPR of the ground truth by at least 60% for inferred trees.
```

Comment:
But the IE line in the other figure shows that a lot of extended stuff does not match also. So yes - this is for simulated trees; 
```suggestion
These measures show that the vast majority of these extended haplotypes are correct if the trees are correct,
```